build-system/tasks/e2e/index.js

```diff
@@ -95,9 +113,11 @@ async function e2e() {
 
     // end task
     if (failures) {
+      process.exit(1);
```

Comment:
This will kill the process immediately, so the promise will never return. Do you absolutely need to add this, given that you're returning `rejecter()`?

Reply:
Yup! I don't need the `rejecter()` actually, but I do need the `process.exit(1)` eventually when I run this in Travis with `pr-check`.